include/timer/timer_config.h

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Eventually... yes.  But this is a large scale refactor that I did not want to get into now as already this got a bit out of scope (but was necessary for some of the new api methods related to the config).